### DISCUSSION TITLE
Radio button border doesn't change?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#5160] [Q&A] [answered]
So far as I can see (tested with the theme creator on the docs), it's impossible to change the border colour of radio buttons in their default state.

It stays light grey, regardless of what colour you set to `buttonBorderColor`.

This can be verified via the theme editor on the [radio button docs page](https://www.naiveui.com/en-US/os-theme/components/radio).

**Comment by Volankey** [comment, marked as answer]
See https://codesandbox.io/p/sandbox/reverent-bhabha-5755lq?file=%2Fsrc%2FDemo.vue%3A8%2C18
